### ISSUE TITLE
support multi gpu

### ISSUE THREAD

**Issue by mkh48v** [omnis-labs-company/omnis#47]

**Comment by 2013-11390** [comment]
We have to change mask rcnn parrallel init function. (We have to import custom mrcnn not original one)

**Comment by 2013-11390** [comment]
In densenet classification model, in same batch size, they didn't have difference in speed but when we use 2 gpus we can train in double batch size.
Train caltech images with 1 gpu with batch size 32 : 83s was taken for 1 epoch
Train caltech images with 2 gpu with batch size 32 : 90s was taken for 1 epoch
Train caltech images with 2 gpu with batch size 64 : 76s was taken for 1 epoch
Small batch size with multiple gpu can take more secs because of merging datas.

**Comment by 2013-11390** [comment]
Keras multi gpu issue: https://github.com/keras-team/keras/issues/9204
Check multi gpu in simple test code, 
Batch size with 128 took 155secs when 2gpu with batch size 256 took 146secs
https://github.com/keras-team/keras/issues/9502
In this issue, they said that keras is slower than other frameworks when using multi gpu

**Comment by 2013-11390** [comment]
In Style Transfer generating in multi gpu model works well in 9f57ddd1a4ee1f7b69feb7c16fde6fc111cf97ce

**Comment by 2013-11390** [comment]
I'll close this issue and make multi gpu issue for each applications such as #299
